src/check-custom-fields.ts

```diff
@@ -0,0 +1,41 @@
+import * as core from '@actions/core'
+
+interface ObjectType {
+  [key: string]: string
+}
+
+interface OptionType {
+  id: string
+  name: string
+}
+
+interface ArrayType {
+  options: OptionType[]
+  name: string
+  value: string
+  id: string
+}
+
+export async function checkDictionaryInArray(obj: {[key: string]: string}, arr: []) {
+  const object: ObjectType = obj
+  const array: ArrayType[] = arr
+
+  const result = []
+
+  for (const key in object) {
+    if (object.hasOwnProperty(key)) {
```

Comment:
Do we really need this one? Iteration should work for all keys without an issue.